### PR TITLE
Add is_primary bool to function hierarchic vl

### PR DESCRIPTION
**Pull request by cymed**
Many functionalities such as SWMM or MIKE exports allow filtering elements of the primary wastewater network. Up to now, all those scripts hardcode a list of primary entries in tww_vl.channel_function_hierarchic, making it cumbersome for people that use vl extensions. To facilitate the exports (and make them quicker), I suggest adding a datamodel extensions in the vl table.

### CODE REVIEW

File: datamodel/changelogs/0001/05_data_model_extensions.sql [hunk 1]
```diff
@@ -53,6 +53,17 @@ SET order_fct_hierarchic=
 		 ]
 	 ,code);
 
+-- this column is an extension to the VSA data model and facilitates filtering out primary features
+ALTER TABLE tww_vl.channel_function_hierarchic ADD COLUMN is_primary bool;
```
Comment (sjib) [inline datamodel/changelogs/0001/05_data_model_extensions.sql:57]: @cymed With our naming convention it should be _is_primary - isn't it?

File: datamodel/changelogs/0001/05_data_model_extensions.sql [hunk 2]
```diff
@@ -53,6 +53,17 @@ SET order_fct_hierarchic=
 		 ]
 	 ,code);
 
+-- this column is an extension to the VSA data model and facilitates filtering out primary features
+ALTER TABLE tww_vl.channel_function_hierarchic ADD COLUMN _is_primary bool;
```
Comment (3nids) [inline datamodel/changelogs/0001/05_data_model_extensions.sql:57]: this needs a default value, you don't want NULL booleans

**Comment by sjib** [comment]
@cymed Would also good to add the DEFAULT FALSE in the adaptions you made to tww_od.organisation see https://github.com/teksi/wastewater/pull/150/files

ALTER TABLE tww_od.organisation ADD COLUMN _active bool;
ALTER TABLE tww_od.organisation ADD COLUMN _local_extension bool;

File: datamodel/changelogs/0001/05_data_model_extensions.sql (hunk 1 above)
Comment (cymed) [reply datamodel/changelogs/0001/05_data_model_extensions.sql]: or tww_is_primary (see https://github.com/teksi/wastewater/pull/164)

**Comment by sjib** [comment]
Plugin test error https://github.com/teksi/wastewater/actions/runs/7898133040/job/21554937421?pr=70
should be solved with https://github.com/teksi/wastewater/pull/178 also

**Comment by cymed** [comment]
Tagging @sjib for review

File: datamodel/changelogs/0001/05_data_model_extensions.sql (hunk 1 above)
Comment (sjib) [reply datamodel/changelogs/0001/05_data_model_extensions.sql]: @cymed then let's do it like in #164 - then we are consistent

**Review by sjib — CHANGES_REQUESTED** [review]
tww_is_primary instead of is_primary to be consistent with #164

**Review by sjib — APPROVED** [review]

**Pull request merged**